### PR TITLE
clientv3: set Serializable from Op

### PR DESCRIPTION
**Pull request by gyuho**
Fix https://github.com/coreos/etcd/issues/4809.

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @xiang90 @heyitsanthony 

Now read latency is <1ms:

```
etcd linearized read
etcd Put started: key_2016-03-18 22:49:35 | value_2016-03-18 22:49:35
etcd Put took 1.116081ms
etcd Get started
etcd Get took 2.670423ms
key_2016-03-18 22:49:35 : value_2016-03-18 22:49:35


etcd serialized read
etcd Put started: key_2016-03-18 22:49:35 | value_2016-03-18 22:49:35
etcd Put took 1.566613ms
etcd Get started
etcd Get WithSerializable took 464.269µs
key_2016-03-18 22:49:35 : value_2016-03-18 22:49:35
```

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
thanks. will merge when the merge-freeze is over.

**Comment by xiang90** [comment]
Ah... that is why my benchmark works before... (it did not use v3client)

**Comment by heyitsanthony** [comment]
I think `toRequestUnion` needs this too

**Comment by gyuho** [comment]
@heyitsanthony I see. Will fix that in the same commit. thanks.

File: clientv3/op.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ type Op struct {
 func (op Op) toRequestUnion() *pb.RequestUnion {
 	switch op.t {
 	case tRange:
-		r := &pb.RangeRequest{Key: op.key, RangeEnd: op.end, Limit: op.limit, Revision: op.rev}
+		r := &pb.RangeRequest{Key: op.key, RangeEnd: op.end, Limit: op.limit, Revision: op.rev, Serializable: op.serializable}
```
Comment (gyuho) [inline clientv3/op.go]: just added this too. thanks.

**Comment by xiang90** [comment]
LGTM. defer to @heyitsanthony

**Comment by gyuho** [comment]
@heyitsanthony Merging? I don't see anywhere else missing this.

Thanks.

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
thanks.

**Pull request merged**